google-cloud-bigquery/lib/google/cloud/bigquery.rb

```diff
@@ -27,11 +27,12 @@ module Cloud
     # BigQuery?](https://cloud.google.com/bigquery/what-is-bigquery).
     #
     # The goal of google-cloud is to provide an API that is comfortable to
-    # Rubyists. Authentication is handled by {Google::Cloud#bigquery}. You can
-    # provide the project and credential information to connect to the BigQuery
-    # service, or if you are running on Google Compute Engine this configuration
-    # is taken care of for you. You can read more about the options for
-    # connecting in the [Authentication
+    # Rubyists. Your authentication credentials are detected automatically in
```

Comment:
Does this change need to be applied to any other gems?

Reply:
Yes. But how does it look?

> Please let me know if the proposed changes to google-cloud-bigquery look good. Once approved, I will copy these changes to other packages and update the PR.

Reply:
Oops, I somehow missed your comment on the issue before I commented. Looks great to me. :+1: